src/main/java/com/google/devtools/build/lib/vfs/OutputPermissions.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 The Bazel Authors. All rights reserved.
```

Comment:
Please remove this file. It's not relevant with this PR.

Reply:
Done